core/chains/evm/client/errors.go

```diff
@@ -666,7 +677,7 @@ var drpc = ClientErrors{
 
 // Linkpool, Blockdaemon, and Chainstack all return "request timed out" if the log results are too large for them to process
 var defaultClient = ClientErrors{
-	TooManyResults: regexp.MustCompile(`request timed out`),
+	TooManyResults: regexp.MustCompile(`request timed out|408 Request Timed Out`),
```

Comment:
Noticed this is different than the error above `408 Request Timeout`. Just wanted to make sure if this was intentional since I don't have too much context on the error

Reply:
Yeah it's intentional, this was requested by Domino here :)